README.md

```diff
@@ -1,3 +1,120 @@
+
+If not, then it may take some extra work on your part. 
+To run a Scala Spark job on AWS you need to compile a fat jar that contains your source code for your job and all of the libraries it needs to run.
+This project already has the [sbt-assembly](https://github.com/sbt/sbt-assembly) plugin setup and a [`assemblyMergeStrategy`](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/build.sbt#L13) set up to package the Spark, Hadoop, and AWS SDK together in the fat jar.
+If you need to add in other libraries that do not play well with each other, or you are using a noncompatible version of Spark for this current repo, there are a few good [resources](http://queirozf.com/entries/creating-scala-fat-jars-for-spark-on-sbt-with-sbt-assembly-plugin) available to help you through the needed `build.sbt` changes.
+
+Outside of the needed changes to [SparkJob.scala](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala) and maybe [build.sbt](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/build.sbt), you need to set a few parameters in the [deploy.sh](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh) script. 
+Mainly the [deploymentPath](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh#L2) to your specific S3 bucket, adding a profile to the [AWS CLI command](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh#L14) to upload to your specific S3 bucket if it's private, and changing the resulting fat jar [name](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh#L3) if you please. 
+
+## Setting Up The Job On AWS
+
+Once you have your fat jar uploaded to S3 its time to set up the scheduled job on AWS Data Pipeline.
+
+#### Create AWS Data Pipeline
+
+Log on to the AWS Dash and make and navigate to the [AWS Data Pipeline](https://console.aws.amazon.com/datapipeline) console click the `Create new pipeline` button.
+
+![alt text][create-pipeline-button]
+
+#### Load The Spark Job Template Definition
+
+Add a name for your pipeline and select the `Import a definition` source option. 
+Included in the [github repo](https://github.com/jessebrizzi/spark-fat-jar-template) for the spark template is a [datapipeline.json](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/datapipeline.json) file that you can import in that contains a pre-defined data pipeline for your spark job that should simplify the setup processes for you.
+The definition contains the node configuration to fire the pipeline off on a schedule and notify you via a [AWS SNS](https://aws.amazon.com/sns/) Alarm on a success or failure run.
+All of your needed configuration options have been parameterized to simplify this process. 
+
+![alt text][create-pipeline-name]
```

Comment:
what does `alt text` do?

Reply:
Its the alt text for the images, i need to set them.